client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ContentLayout.tsx

```diff
@@ -69,11 +77,38 @@ export const NumInputRow = ({
     }
   };
 
+  const valueInDoses = React.useMemo(
+    () =>
+      displayVaccinesInDoses
+        ? calculateValueInDoses(
+            representation,
+            defaultPackSize,
+            dosesPerUnit,
+            valueInUnitsOrPacks
+          )
+        : undefined,
+    [
+      displayVaccinesInDoses,
+      representation,
+      defaultPackSize,
+      dosesPerUnit,
+      valueInUnitsOrPacks,
+    ]
+  );
+
   return (
-    <Box sx={{ marginBottom: 1, px: 1, flex: 1, ...sx }}>
+    <Box
+      sx={{
+        marginBottom: 1,
+        px: 1,
+        flex: 1,
+        ...sx,
+      }}
+    >
       <InputWithLabelRow
         Input={
           <NumericTextInput
+            fullWidth
```

Comment:
Don't think this is good use of space, esp with keyboard... I think it fit fine when they were all on one line??

![Screenshot 2025-06-05 at 10 35 45 AM](https://github.com/user-attachments/assets/c3bdfb96-25f0-4068-b124-21adebf8bf91)
![Screenshot 2025-06-05 at 10 36 27 AM](https://github.com/user-attachments/assets/1d6c6784-a0d8-48d5-8758-125c1194d89b)

Reply:
Lol, I think I'd say if we're supporting vertical tablet, we shouldn't have 3 columns 👀  Okay to leave for now, should probs make further changes after proper design review/input

Reply:
3 columns is not standard. Currently only for 1 client only. Can think of this later if it does expand..